sources/packages/backend/apps/api/src/route-controllers/application/application.aest.controller.ts

```diff
@@ -117,4 +124,65 @@ export class ApplicationAESTController extends BaseController {
       throw error;
     }
   }
+
+  /**
+   * Triggers manual reassessment for an application.
+   * Application cannot be archived and current assessment must be in completed status.
+   * @param applicationId application id.
+   */
+  @Roles(Role.AESTAdmin)
+  @Post(":applicationId/manual-reassessment")
```

Comment:
I am not sure if this endpoint belongs here or under `assessment.aest.controller.ts`.
@dheepak-aot @guru-aot @lewischen-aot @sh16011993 thoughts?

Reply:
IMO, the endpoint must come under `assessment.aest.controller`